src/style.css

```
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.any__star {
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .any__star:nth-of-type(1) {
+  background: url(images/star-active.svg) no-repeat center;
```

You don't need to repeat these properties. You've already set them above in your code.
```suggestion
  background: url(images/star-active.svg);
```